utils.py

```diff
@@ -88,11 +108,13 @@ def get_file_contents(filepath, raw_bytes=False, mode='r'):
     else:
         encoding = 'utf-8'
 
-    with python_utils.open_file(filepath, mode, encoding=encoding) as f:
-        return f.read()
+    with python_utils.open_file( # type: ignore[no-untyped-call]
+        filepath, mode, encoding=encoding) as f:
+        return f.read() # type: ignore[no-any-return]
```

Comment:
Why is this ignore needed?

Reply:
The python_utils.open_file() function is not type annotated, so the return type of such a function is `Any'. So f.read() is of type any too. So the return type of the method we are writing is Any. And we have specified in configurations file `warn-return-any` as True. An error will be thrown when returning `Any`. So to ignore this error, we have added type ignore.